apps/agent/buidler.config.js

```diff
@@ -0,0 +1,51 @@
+const { usePlugin } = require('@nomiclabs/buidler/config')
+
+usePlugin("@nomiclabs/buidler-ganache")
+usePlugin('@nomiclabs/buidler-truffle5')
+usePlugin('buidler-gas-reporter')
+usePlugin('solidity-coverage')
+
+module.exports = {
+  networks: {
+    coverage: {
+      url: 'http://localhost:8555',
+    },
+    mainnet: {
```

Comment:
Perhaps we can leave a few quick comments about the development networks (`ganache` and `coverage`) and the real ones?